src/worker.js

```diff
@@ -54,6 +54,10 @@ const initServer = (app, config, callback) => {
       .catch(exit(code));
   }
 
+  function readyToClose(getClose) {
+    getClose(shutDownSequence);
```

Comment:
I don't think we want to delegate the responsibility of closing down the server to a callback. We should close and then fire off a callback when it's done.